pytraj/externals/_json.py

```diff
@@ -43,5 +43,5 @@ def try_read(path):
             with open(path, mode) as fh:
                 return json.load(fh)
         except (Exception) as e:
-            raise
+            raise e
```

Comment:
Why did you make this change?  With `raise`, you get the original traceback (which is useful).  If you change it to `raise e`, you get the traceback that points you back to line 46 (not nearly as useful).

Also, since you are promptly re-raising the exception you caught, you'd simplify your code by just omitting the the `try ... except` altogether.